src/session-manager/index.ts

```diff
@@ -13,7 +14,7 @@ abstract class SessionManager {
    */
   public static async logout(options?: ConfigOptions): Promise<Response> {
     const { realmPath, serverConfig } = Config.get(options);
-    const init: RequestInit = {
+    let init: RequestInit = {
```

Comment:
This looks like it also could be a single line const.

Reply:
Do you mean just inline this init object into the middlewareWrapper call below?

Reply:
yep just like that.